### PR TITLE
Deprecates Sans/Serif; mixes in textTransform to Text

### PR DESCRIPTION
**Pull request by dzucconi**
The deprecation was long overdue. :)

### CODE REVIEW

File: packages/palette/src/elements/Text/Text.tsx [hunk 1]
```diff
@@ -29,8 +30,18 @@ const textColor = style({
   key: "colors",
 })
 
+type TextTransform =
+  | "none"
+  | "capitalize"
+  | "uppercase"
+  | "lowercase"
+  | "initial"
+  | "inherit"
```
Comment (damassi) [inline packages/palette/src/elements/Text/Text.tsx:39]: Could this be pulled from general dom types? `CSS.Something`

Comment (dzucconi) [reply packages/palette/src/elements/Text/Text.tsx:39]: I can if I pull in https://github.com/frenic/csstype — it's a dependency of styled-system anyway 🤷

Comment (damassi) [reply packages/palette/src/elements/Text/Text.tsx:39]: Oh i see, i was thinking that was a [part of this](https://github.com/artsy/force/blob/master/tsconfig.json#L23)

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette@14.7.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4014.7.0) :rocket:

**Pull request merged**